src/libs/tagging.c

```diff
@@ -1387,6 +2223,8 @@ int position()
 
 void gui_init(dt_lib_module_t *self)
 {
+  setvbuf(stdout, NULL, _IONBF, 0);
```

Comment:
This line and the following looks like debug ones. Should be removed;

Reply:
done